documentation/src/test/java/org/hibernate/search/documentation/mapper/orm/indexing/HibernateOrmBatchJsr352IT.java

```diff
@@ -56,13 +56,13 @@ public void setup() {
 	@Test
 	public void simple() throws Exception {
 		// tag::simple[]
-		Properties jobProps = MassIndexingJob
+		Properties jobProps = MassIndexingJob // <1>
 				.parameters()
```

Comment:
```suggestion
		Properties jobProps = MassIndexingJob.parameters() // <1>
```

Reply:
Did you miss this suggestion or decide against applying it?

Reply:
Actually I thought I applied for the change... Doing now...